int8 ACK=1
int8 DONE=2
int8 EMPTY=3 

Header header
int8 code
int32 queue_length
float32 queue_time
int32[6] positions
int32[6] planner_positions
